R/model-summary.R

```diff
@@ -12,6 +12,9 @@
 #' **NONMEM**
 #'
 #' The returned list for a NONMEM model will contain the following top-level elements:
+#'  * **absolute_model_path** -- Absolute path to the model that generated this summary.
+#'    Note, using this directly is discouraged in favor of using the "getters"
+#'    described in `?get_path_from_object`.
```

Comment:
Link to documentation rather than `?get_path_from_object`.

Reply:
same as https://github.com/metrumresearchgroup/rbabylon/pull/297/files#r545953884